.github/workflows/pullrequest.yml

```diff
@@ -34,7 +34,7 @@ jobs:
             ${{ runner.os }}-maven-
 
       - name: Build with Maven
-        run: mvn --batch-mode --update-snapshots verify -P integration-test
+        run: mvn --batch-mode --update-snapshots verify # -P integration-test - add this back once we have a compatible flagd
```

Comment:
Removing the `integration-test` profile disables the integration test suite as discussed in the description.